backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/LocalDateFieldSortBuilderFactory.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.types.sort.impl;
+
+import java.lang.invoke.MethodHandles;
+import java.time.LocalDate;
+
+import org.hibernate.search.backend.lucene.logging.impl.Log;
+import org.hibernate.search.backend.lucene.search.sort.impl.LuceneSearchSortBuilder;
+import org.hibernate.search.backend.lucene.types.converter.impl.LuceneFieldConverter;
+import org.hibernate.search.engine.logging.spi.EventContexts;
+import org.hibernate.search.engine.search.sort.spi.DistanceSortBuilder;
+import org.hibernate.search.engine.search.sort.spi.FieldSortBuilder;
+import org.hibernate.search.engine.spatial.GeoPoint;
+import org.hibernate.search.util.impl.common.LoggerFactory;
+
+public class LocalDateFieldSortBuilderFactory implements LuceneFieldSortBuilderFactory {
+
+	private static final Log log = LoggerFactory.make( Log.class, MethodHandles.lookup() );
+
+	private final LuceneFieldConverter<LocalDate, ?> converter;
+
+	public LocalDateFieldSortBuilderFactory(LuceneFieldConverter<LocalDate, ?> converter) {
+		this.converter = converter;
+	}
+
+	@Override
+	public FieldSortBuilder<LuceneSearchSortBuilder> createFieldSortBuilder(String absoluteFieldPath) {
+		return new LocalDateFieldSortBuilder( absoluteFieldPath, converter );
+	}
+
+	@Override
+	public DistanceSortBuilder<LuceneSearchSortBuilder> createDistanceSortBuilder(String absoluteFieldPath,
```

Comment:
You could have an `AbstractStandardLuceneFieldSortBuilderFactory`, similarly to the `AbstractStandardLuceneFieldPredicateBuilderFactory`, that would implement the the `converter` field, and the `createDistanceSortBuilder` and `isDslCompatibleWith` methods. Not a big improvement, but that would remove a little bit of duplicated code.